catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookByIndexBrick.java

```diff
@@ -67,24 +66,12 @@ public int getViewResource() {
 	@Override
 	public View getView(Context context) {
 		super.getView(context);
-
-		if (getSprite().getName().equals(context.getString(R.string.background))) {
-			TextView label = view.findViewById(R.id.brick_set_look_by_index_label);
-			label.setText(R.string.brick_set_background_by_index);
-		}
-
 		return view;
 	}
 
 	@Override
 	public View getPrototypeView(Context context) {
 		View prototypeView = super.getPrototypeView(context);
-
```

Comment:
It is not necessary to override a method that just calls `super` and returns its result. You can just remove the entire method `public View getPrototypeView(Context context) {..}`